pkgs/os-specific/linux/system76-io/default.nix

```diff
@@ -27,12 +27,13 @@ stdenv.mkDerivation {
     install -D system76-io.ko $out/lib/modules/${kernel.modDirVersion}/misc/system76-io.ko
   '';
 
-  meta = with lib; {
-    maintainers = [ maintainers.khumba ];
-    license = [ licenses.gpl2Plus ];
-    platforms = [ "i686-linux" "x86_64-linux" ];
-    broken = versionOlder kernel.version "4.14";
-    description = "DKMS module for controlling System76 I/O board";
+  meta = {
+    maintainers = with lib.maintainers; [ khumba ahoneybun ];
+    license = lib.licenses.gpl2Plus;
+    platforms = [ "i686-linux" "x86_64-linux" "aarch64-linux" ];
+    broken = lib.versionOlder kernel.version "5.10";
+    description = "DKMS module for controlling System76 Io board";
     homepage = "https://github.com/pop-os/system76-io-dkms";
   };
 }
+
```

Comment:
Did you mean to commit this white space?

Reply:
Should be fixed now.